packages/api/src/graphql/event/event.ts

```diff
@@ -0,0 +1,95 @@
+import {Event, EventStatus} from '@prisma/client'
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLID,
+  GraphQLInputObjectType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString
+} from 'graphql'
+import {GraphQLDateTime} from 'graphql-iso-date'
+import {Context} from '../../context'
+import {ConnectionResult} from '../../db/common'
+import {createProxyingResolver} from '../../utility'
+import {GraphQLPageInfo} from '../common'
+import {GraphQLImage} from '../image'
+import {GraphQLTag} from '../tag/tag'
+import {EventSort} from './event.queries'
+import {GraphQLRichText} from '../richText'
+
+export const GraphQLEventStatus = new GraphQLEnumType({
+  name: 'EventStatus',
+  values: {
+    CANCELLED: {value: EventStatus.Cancelled},
+    RESCHEDULED: {value: EventStatus.Rescheduled},
+    POSTPONED: {value: EventStatus.Postponed},
+    SCHEDULED: {value: EventStatus.Scheduled}
+  }
+})
+
+export const GraphQLEvent = new GraphQLObjectType<Event, Context>({
+  name: 'Event',
+  fields: {
+    id: {type: GraphQLNonNull(GraphQLID)},
+
+    name: {type: GraphQLNonNull(GraphQLString)},
+    status: {type: GraphQLNonNull(GraphQLEventStatus)},
+    description: {type: GraphQLRichText},
+
+    location: {type: GraphQLString},
+    startsAt: {type: GraphQLNonNull(GraphQLDateTime)},
+    endsAt: {type: GraphQLDateTime},
+
+    tags: {
+      type: GraphQLList(GraphQLNonNull(GraphQLTag)),
+      resolve: createProxyingResolver(async ({id}, _, {prisma: {tag}}) => {
```

Comment:
Could we avoid implementing new proxying resolvers and implement includes (in Prisma terms, or joins in sql terms) instead?

Reply:
That's technically an anti pattern in GraphQL, as if you do not care about tags now you do extra database calls.